parameter/ParameterMgr.cpp

```diff
@@ -88,11 +87,9 @@
 #include "XmlStringDocSource.h"
 #include "XmlMemoryDocSink.h"
 #include "XmlMemoryDocSource.h"
-#include "SelectionCriteriaDefinition.h"
+#include "CommandParser.h"
```

Comment:
would it be possible to have something like command/Parser.h ? It will make things easier when command eventually become a library.